### ISSUE TITLE
Question about NUCLA dataset

### ISSUE THREAD

**Issue by sydat2701** [srijandas07/VPN#11]
Hi, could you please give me the list of videos for the training set and validation set (cross-view method)? I found that the original NUCLA dataset has 1463 videos, but in your paper, there are only 1194. 
Thank you for your help!

**Comment by srijandas07** [comment]
Sorry for the delay in responding. I have uploaded the splits. We have used train_cross3.txt for training and test_cross3.txt for testing.

**Comment by srijandas07** [comment]
@sydat2701 I am closing the issue now.
